LongWarOfTheChosen/Config/XComGameData_WeaponData.ini

```diff
@@ -2178,10 +2119,10 @@ TEMPLAR_PISTOL_RANGE[23] = -80
 TEMPLAR_PISTOL_RANGE[24] = -90
 TEMPLAR_PISTOL_RANGE[25] = -100
 
-+Bullpup_Laser_ImagePath="UILibrary_LW_LaserPack.LaserSMG_Base"
-+Bullpup_Coil_ImagePath="UILibrary_LW_Overhaul.InventoryArt.CoilSMG_Base"
-+Vektor_Laser_ImagePath="CrossbowVektor.Inv_CrossbowVektor"
-+Vektor_Coil_ImagePath="CrossbowVektor.Inv_CrossbowVektor"
+Bullpup_Laser_ImagePath="UILibrary_LW_LaserPack.LaserSMG_Base"
+Bullpup_Coil_ImagePath="UILibrary_LW_Overhaul.InventoryArt.CoilSMG_Base"
```

Comment:
Wait, THAT's what fixes the faction weapon icons?

Reply:
If you're talking about the removal the of `+` at the start of the line, no. That's just my obsessive removal of unnecessary `+`s. The fix for faction weapon icons is simply using the correct image path for inventory, for example on [line 729](https://github.com/long-war-2/lwotc/pull/954/files#diff-1a46b2cb210700088dfc5f63191b7e77R729). It looks like the laser and coil versions of the Bullpup are already configured correctly.